R/latent_individual.R

```diff
@@ -108,28 +108,81 @@ is_latent_individual <- function(data) {
   inherits(data, "epidist_latent_individual")
 }
 
+#' Check if data has the `epidist_latent_individual` class
+#'
+#' @param data A `data.frame` or `data.table` containing line list data
+#' @param family Output of a call to `brms::brmsfamily()`
+#'
+#' @importFrom rstan lookup
+#' @method epidist_family epidist_latent_individual
+#' @family latent_individual
+#' @export
+epidist_family.epidist_latent_individual <- function(data,
+                                                     family = brms::lognormal(),
+                                                     ...) {
+  epidist_validate(data)
+  if (!checkmate::test_class(family, "family")) {
+    cli::cli_abort("family must be a family object")
+  }
+  family <- brms:::validate_family(family) # allows use of stats:: family
+  # brms also has lb and ub in brms::custom_family, but not in brms::brmsfamily
+  non_mu_links <- family[[paste0("link_", setdiff(family$dpars, "mu"))]]
+  # Use the function dpar_bounds() here
```

Comment:
Need to try using this function to get the bounds. It does work though as is, so what's the point? Is there a way to let user give this as input within `brmsfamily` objects?

Reply:
Have done this but still have questions